content/posts/2025/01/teaching-llms-report.md

```diff
@@ -0,0 +1,98 @@
+Most participants were Carpentries Instructors, familiar with the challenge of fitting the current lesson content into the time available at a two-day workshop. 
+How would we make space to teach LLM assistants effectively in workshops that are already packed to bursting with other important content?
+
+
+## LLM assistants and The Carpentries Core Values
+Central to the discussion in both community sessions was the question of how teaching LLMs in Carpentries workshops might align with or against [the community's Core Values](/about-us/#our-values). 
+Participants highlighted concerns about the lack of attribution of the data used to train the models (conflicting with our commitments to act openly and value all contributions), and the significant energy and resource requirements leading to environmental impact of the training process itself. 
+
+On the other hand, adapting The Carpentries lessons to teach about LLMs and how to use them effectively would align with our commitment to empower one another and our eagerness to keep learning.
+
+Finally, the aforementioned potential difficulties with accessing LLM assistants in different institutions, and concerns about the cost of using these tools in the future, were identified as potentially conflicting with the commitment to championing _access for all_.
+
+
+## Support for teaching LLM assistants in Carpentries workshops
+While acknowledging these concerns, the predominant view among participants at the community sessions was that The Carpentries should be teaching LLM assistants in workshops, alongside the fundamental concepts and good practices that we already cover.
+Learners need a working mental model of the domain to be able to interpret answers and debug code suggestions provided by an LLM, and need to be informed about what LLMs are and are not good at, the kinds of tasks they should and should not be used for, and the common difficulties they may face when using them.
+Support among participants was divided evenly between the suggested options of teaching these tools only as a section within existing lessons, or in more detail in a dedicated lesson/workshop.
+
+
+## Conclusion and trailer for part 2
+These were excellent community discussions: another reminder that The Carpentries community has so much valuable experience and expertise, a willingness to share it, and a desire to learn from one another. 
+We are so grateful to all the community members who joined the sessions and participated in the discussions -- **thank you!** 
+We look forward to continuing the conversation in 2025. 
+
+In part 2, we will summarise how the Curriculum Team plans to respond to the points raised in these two community discussions. 
+One element of that response will be several more community discussions, around the same theme of _AI and The Carpentries_, in the first few months of the year (follow links for events times in your local time zone):
```

Comment:
```suggestion
One element of that response will be follow-up discussions, around specific topics within the theme of _AI and The Carpentries_, in the first few months of the year (follow links for events times in your local time zone):
```

Reply:
I changed the wording here to try to avoid it sounding like we were going to just keep talking about the same thing forever and not do anything. . . .